.env.test

```
@@ -0,0 +1,10 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+
+# trello authorize
+TRELLO_TOKEN=fwef23423423wegwegwegwgw3f0c4d1wefweufn23i45h23432423fnwfc2B5EE9058
+TRELLO_KEY=234gwefgwegwegergwe34r234ngwejgwgwg3tw3t
```

I can tell when I look a bit harder that these are clearly fake, but at first glance I still panicked! I'm a big fan of _really_ obviously fake name so that future devs don't have to worry about whether or not it might be a security concern, and if someone _does_ accidentally upload real ones it will be really obvious
```suggestion
TRELLO_TOKEN=this-is-a-fake-token
TRELLO_KEY=this-is-a-fake-key
```